sapl/comissoes/views.py

```diff
@@ -249,6 +280,98 @@ def get_initial(self):
             return {'comissao': comissao}
 
 
+class RemovePautaView(PermissionRequiredMixin, CreateView):
+    model = PautaReuniao
+    form_class = PautaReuniaoForm
+    template_name = 'comissoes/pauta.html'
+    permission_required = ('comissoes.add_reuniao', )
+
+    def get_context_data(self, **kwargs):
+        context = super(
+            RemovePautaView, self
+        ).get_context_data(**kwargs)
+
+        # Remove = 0; Adiciona = 1
+        context['opcao'] = 0
+
+        context['object'] = Reuniao.objects.get(pk=self.kwargs['pk'])
+        context['root_pk'] = context['object'].comissao.pk
+
+        materias_pauta = PautaReuniao.objects.filter(reuniao=context['object'])
+        materias_pk = [materia_pauta.materia.pk for materia_pauta in materias_pauta]
+        
+        context['materias'] = MateriaLegislativa.objects.filter(
+            pk__in=materias_pk
+        ).order_by('tipo', '-ano', '-numero') 
+        context['num_materias'] = len(context['materias'])
+
+        return context
+
+    def post(self, request, *args, **kwargs):
+        success_url = reverse('sapl.comissoes:reuniao_detail', kwargs={'pk':kwargs['pk']})
+        marcadas = request.POST.getlist('materia_id')
+
+        if len(marcadas) == 0:
+            msg=_('Nenhuma matéria foi selecionada.')
+            messages.add_message(request, messages.WARNING, msg)
+            return HttpResponseRedirect(success_url)
+
+        reuniao = Reuniao.objects.get(pk=kwargs['pk'])
+        for materia in MateriaLegislativa.objects.filter(id__in=marcadas):
+            PautaReuniao.objects.get(reuniao=reuniao,materia=materia).delete()
```

Comment:
Eu substituiria por um `filter`, pois no caso de não existir PautaReuniao ele não dá erro de `ObjectDoesNotExist`
```suggestion
            PautaReuniao.objects.filter(reuniao=reuniao,materia=materia).delete()
```